src/controllers/dictionaryController.ts

```diff
@@ -23,10 +23,11 @@ import { BadRequestError } from '../utils/errors';
 import { replaceReferences } from '../utils/references';
 import { diff as diffUtil } from '../diff/DictionaryDiff';
 import logger from '../config/logger';
+import { DictionaryDocument } from '../models/Dictionary';
 
 export const listDictionaries = async (req: Request, res: Response) => {
-  const name = req.query.name;
-  const version = req.query.version;
+  const name = req.query.name.toString();
+  const version = req.query.version.toString();
```

Comment:
express added more typing `req.query` can be a string array or other specific types. using `.toString()` we always get a string then our actually functions can check if its a valid dictionary, version etc. Seems more logically to me than just casting to a string